### DISCUSSION TITLE
New source: Vertica

### DISCUSSION THREAD

**Discussion by rmuthu** [airbytehq/airbyte#36088] [New Connector Request]
## Tell us about the new connector you’d like to have
* Source-vertica and destination-iceberg connector

**Comment by harshithmullapudi** [comment]
Hey, could you explain more detail on this issue? Is this about new source vertica?

**Comment by rmuthu** [comment]
Yes. source connector for vertica

On Mon, Aug 8, 2022 at 10:03 AM Harshith Mullapudi ***@***.***>
wrote:

> Hey, could you explain more detail on this issue? Is this about new source
> vertica?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/airbytehq/airbyte/issues/15391#issuecomment-1207654789>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACXBTBNBGU7FFHCN5BVJX3TVYCEX5ANCNFSM552ZR7SA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Rathika M*

**Comment by hrmnjt** [comment]
+1

**Comment by carlos-verdes** [comment]
Is there any code we can review/help for this?

I can see there is already an announcement for this connector:
https://airbyte.com/connectors/vertica

**Comment by harshithmullapudi** [comment]
Hey I don't think we have any PR's regarding this source yet.

**Comment by carlos-verdes** [comment]
Do you know why it's on the main page then?

**Comment by harshithmullapudi** [comment]
I think it says coming up.

**Comment by carlos-verdes** [comment]
My point is if you know who published the announcement and if we can contact them for collaboration, we started similar effort and it has no sense to not work together :)

**Comment by harshithmullapudi** [comment]
Got it. @misteryeo do you have any information on this?

**Comment by misteryeo** [comment]
@carlos-verdes The page you shared is auto-generated based on our users and customers **requesting** the connector. You'll notice that it says that there are 75 **upvotes**. So no one is actively working on this from what I can see so feel free to make a contribution! 
![airbyte comconnectorsvertica 2022-09-27 at 7 23 56 AM](https://user-images.githubusercontent.com/16851149/192553258-abff5c8f-984e-44b6-a533-39e5ef669781.jpg)
